### PR TITLE
Loader 2

### PR DESCRIPTION
**Pull request by sonals**

### CODE REVIEW

**Comment by anupcs-xlnx** [comment]
Just a reminder that these changes need to be cherry-picked to PU1 as well.

**Review by rozumx — APPROVED** [review]
Looks good.

**Review by maxzhen — APPROVED** [review]

**Review by AShivangi — COMMENTED** [review]
Hi @sonals, 
We most likely won't be able to get xbutil2 reset working in time. Should we remove that and default to xbutil?
If the user runs xbutil2 reset specific options, the script won't find it and will give an error.

File: src/runtime_src/core/tools/xbutil2/xbutil [hunk 1]
```diff
@@ -1,19 +1,134 @@
...
+declare -A PROGOPT=( \
+        ["-p"]="XBUTIL" \
+        ["--user"]="XBUTIL2" \
+        ["-u"]="XBUTIL2" \
+        ["--shell"]="XBUTIL2" \
+        ["-s"]="XBUTIL2" \
+        ["--base"]="XBUTIL2" \
+        ["-b"]="XBUTIL2" \
+        ["--cache"]="XBUTIL2" \
+        ["-c"]="XBUTIL2" \
```
Comment (AShivangi) [inline src/runtime_src/core/tools/xbutil2/xbutil:60]: Aren't these xbmgmt2 options?

Comment (sonals) [reply src/runtime_src/core/tools/xbutil2/xbutil:60]: Sharp eye! Will update.

Comment (sonals) [reply src/runtime_src/core/tools/xbutil2/xbutil:60]: Will also make the change for reset

Comment (sonals) [reply src/runtime_src/core/tools/xbutil2/xbutil]: updated

Comment (AShivangi) [reply src/runtime_src/core/tools/xbutil2/xbutil]: Will you be updating the xbutil reset as well?

**Review by AShivangi — APPROVED** [review]
Thank you :)

File: src/runtime_src/core/tools/xbutil2/xbutil (hunk 1 above)
Comment (sonals) [reply src/runtime_src/core/tools/xbutil2/xbutil]: Done, reset defaults to legacy.

**Comment by xilinxgitops** [comment]
retest this please

**Pull request merged**